lib/wallet/src/Cardano/Wallet.hs

```diff
@@ -2061,7 +2061,7 @@ buildAndSignTransactionPure
             , builtSealedTx = signedTx
             }
   where
-    anyCardanoEra = Write.fromAnyRecentEra era
+    anyCardanoEra = Write.toAnyCardanoEra era
```

Comment:
🤷‍♂️ I think both names are meh. Maybe some `weaken` / `strengthen` class could be had instead...